lib/game.rb

```diff
@@ -0,0 +1,12 @@
+class Game
+  attr_reader :start_game
+
+  def initialize
+    @start_game =  "Welcome to BATTLESHIP"
```

Comment:
This passes our RSpec test and prints out in our pry session, but it does not print in the terminal so we may have to figure out another way to get this to work.

Reply:
Let's try a runner file, and run from there.